A few refinements across the board I (#21)
crate/cloud-docs: docs/tutorials/cluster-deployment/croud.rst

This will create a new project named `sampleproject`.

> Deploy cluster

Reviewer: ```suggestion
Deploying clusters
```

   sh$ croud clusters deploy --product-name cr1 --tier default --cluster-name my-crate-cluster --project-id f76d96aa-f1a7-46aa-a89b-8cdd2b3cef15 --version 4.8.0 --username admin --password "as6da9ddasfaad7i902jcv780dmcba" --subscription-id 3a35974f-5319-47fb-9a1f-ab85dca75c86

Scale cluster
=============

> After deploying, the next thing that you might want to do is scale your
> cluster. This allows you to either add more nodes or more storage to your

Reviewer: ```suggestion
Clusters can be scaled at any time, this allows you to add more nodes
or more storage to your
```
